localplanning.services/src/components/LpaMasthead.astro

```diff
@@ -0,0 +1,39 @@
+---
+import Container from "@components/Container.astro";
+import { Image } from "astro:assets";
+
+interface Props {
+  title: string;
+  lpaName: string;
+  backgroundColor?: string;
```

Comment:
`backgroundColor` is never passed in as a prop, and will always just be `bg-bg-main`. Maybe we can just simplify this on line 17?